doonut-app-external-api/src/main/kotlin/com/doonutmate/image/service/ImageService.kt

```diff
@@ -52,6 +52,7 @@ class ImageService(
             .height(imageMeta.height)
             .width(imageMeta.widht)
             .capacity(imageMeta.capacity)
+            .memberId(imageBusinessService.getMemberByOauthId(memberId).id)
```

Comment:
저희 로그인하면 oauthId가 아닌 memberId가 들어와서 따로 변환할 필요 없지 않나요?  
```suggestion
            .memberId(memberId)
```

Reply:
확인해보니 그렇네요 바꾸어두었습니다!